src/util.ts

```diff
@@ -1,10 +1,30 @@
+import { resolve4, setServers, getServers, LookupAddress, LookupOptions, lookup } from 'dns';
 import { GMT } from './index';
-import { LookupAddress, LookupOptions, lookup } from 'dns';
 
-export function dnsLookup(
+export function resolveDNS(
+	host: string
+): Promise<string[]> {
+	if(process.env.CUSTOM_DNS_SERVER) {
+		let currentDNSservers = getServers();
+		setServers([process.env.CUSTOM_DNS_SERVER, ...currentDNSservers]);
```

Comment:
I'm not a huge fan of the env var approach. I'd prefer if you pass in a property (`dnsServer` or something like that) to the options object when creating the PAC resolver instance.

Reply:
Also I don't think this should be invoked for every `resolveDNS()` invocation. Maybe we do it in the constructor function instead?

Reply:
Sounds good. But how could I access the options in util.ts and which part would you like to move to the constructor function(constructor function you mean index.ts)?